README.md

```diff
@@ -193,7 +193,7 @@ https://github.com/dangreenisrael/eslint-plugin-jest-formatting
 [prefer-to-contain]: docs/rules/prefer-to-contain.md
 [prefer-to-have-length]: docs/rules/prefer-to-have-length.md
 [prefer-todo]: docs/rules/prefer-todo.md
-[require-tothrow-message]: docs/rules/require-tothrow-message.md
```

Comment:
It might be nicer to have both docs, & change `require-tothrow-message` to had "deprecated" + a link to the replacement as it's body.

That way any existing links would keep working, and would save having to change `rules.test` twice.

Reply:
I personally think it is more confusing to have them both listed and is better solved in the back with code. But if this is wanted I'll make the change no problem.